tasks/prepare.yml

```diff
@@ -21,6 +21,7 @@
 - name: update package cache
   ansible.builtin.package:
     update_cache: true
+    cache_valid_time: 1800
```

Comment:
This is only supported exclusively with Debian derivatives, but not with ArchLinux:

```
msg: 'Unsupported parameters for (ansible.legacy.pacman) module: cache_valid_time. Supported parameters include: executable, upgrade, force, extra_args, upgrade_extra_args, remove_nosave, update_cache, update_cache_extra_args, name (package, pkg), state.'
```

Reply:
Sorry, I didn't test on Arch.